lib/Backends/OpenCL/CMakeLists.txt

```diff
@@ -1,30 +1,38 @@
+set(OpenCL_DIR ${CMAKE_BINARY_DIR}/glow/OpenCL)
+
+add_custom_command(
+    OUTPUT ${OpenCL_DIR}
+    COMMAND ${CMAKE_COMMAND} -E make_directory ${OpenCL_DIR})
+add_custom_target(make_opencl_dir
+    DEPENDS ${OpenCL_DIR})
+
 add_custom_command(
-  OUTPUT "${CMAKE_BINARY_DIR}/glow/OpenCL/kernels.cl.inc"
+  OUTPUT "${OpenCL_DIR}/kernels.cl.inc"
   COMMAND include-bin
           "${CMAKE_CURRENT_SOURCE_DIR}/kernels.cl"
-          "${CMAKE_BINARY_DIR}/glow/OpenCL/kernels.cl.inc"
-  DEPENDS include-bin CPURuntime "${CMAKE_CURRENT_SOURCE_DIR}/kernels.cl")
+          "${OpenCL_DIR}/kernels.cl.inc"
+  DEPENDS make_opencl_dir include-bin CPURuntime "${CMAKE_CURRENT_SOURCE_DIR}/kernels.cl")
 
 add_custom_command(
-  OUTPUT "${CMAKE_BINARY_DIR}/glow/OpenCL/kernels_fwd_conv.cl.inc"
+  OUTPUT "${OpenCL_DIR}/kernels_fwd_conv.cl.inc"
   COMMAND include-bin
           "${CMAKE_CURRENT_SOURCE_DIR}/kernels_fwd_conv.cl"
-          "${CMAKE_BINARY_DIR}/glow/OpenCL/kernels_fwd_conv.cl.inc"
-  DEPENDS include-bin CPURuntime
+          "${OpenCL_DIR}/kernels_fwd_conv.cl.inc"
+  DEPENDS make_opencl_dir include-bin CPURuntime
           "${CMAKE_CURRENT_SOURCE_DIR}/kernels_fwd_conv.cl")
 
 add_custom_command(
-  OUTPUT "${CMAKE_BINARY_DIR}/glow/OpenCL/kernels_fwd_quantized_conv.cl.inc"
+  OUTPUT "${OpenCL_DIR}/kernels_fwd_quantized_conv.cl.inc"
   COMMAND include-bin
           "${CMAKE_CURRENT_SOURCE_DIR}/kernels_fwd_quantized_conv.cl"
-          "${CMAKE_BINARY_DIR}/glow/OpenCL/kernels_fwd_quantized_conv.cl.inc"
-  DEPENDS include-bin CPURuntime
+          "${OpenCL_DIR}/kernels_fwd_quantized_conv.cl.inc"
+  DEPENDS make_opencl_dir include-bin CPURuntime
```

Comment:
Not a problem in this PR, but why do we depend on `CPURuntime` in our `OpenCL` CMake file? Is it a copy&paste bug or something like this?

@rdzhabarov @SplitInfinity

Reply:
Hmm, strange, I'm not sure...

Reply:
hehe, no idea, but @opti-mix you introduced that in
https://github.com/pytorch/glow/commit/2b9abbc012437004e6b9766ded2b400903598a07

Reply:
Ha-ha. So, it was me??? :-D I'm almost sure it is a copy&paste error.